### PR TITLE
chore: typos

### PR DESCRIPTION
**Pull request by abarmat**

### CODE REVIEW

File: contracts/land/ILANDRegistry.sol [hunk 1]
```diff
@@ -6,7 +6,7 @@ interface ILANDRegistry {
   function assignNewParcel(int x, int y, address beneficiary) external;
   function assignMultipleParcels(int[] x, int[] y, address beneficiary) external;
 
-  // After one year, land can be claimed from an inactive public key
+  // After one year, Land can be claimed from an inactive public key
```
Comment (NicoSantangelo) [inline contracts/land/ILANDRegistry.sol]: If we're changing this, we might want to use `LAND` ?

File: contracts/estate/EstateStorage.sol [hunk 2]
```diff
@@ -11,13 +11,13 @@ contract LANDRegistry {
...
 
-  // From Estate to list of owned land ids (LANDs)
+  // From Estate to list of owned Land ids (LANDs)
   mapping(uint256 => uint256[]) public estateLandIds;
 
   // From Land id (LAND) to its owner Estate id
   mapping(uint256 => uint256) public landIdEstate;
 
-  // From Estate id to mapping of land id to index on the array above (estateLandIds)
+  // From Estate id to mapping of Land id to index on the array above (estateLandIds)
```
Comment (NicoSantangelo) [inline contracts/estate/EstateStorage.sol]: If we're changing this, we might want to use `LAND` ?

File: contracts/estate/EstateStorage.sol [hunk 3]
```diff
@@ -11,13 +11,13 @@ contract LANDRegistry {
 contract EstateStorage {
   LANDRegistry public registry;
 
-  // From Estate to list of owned land ids (LANDs)
+  // From Estate to list of owned Land ids (LANDs)
```
Comment (NicoSantangelo) [inline contracts/estate/EstateStorage.sol]: If we're changing this, we might want to use `LAND` ?

**Comment by abarmat** [comment]
Agree on your comments.

**Pull request merged**